src/components/Grid/Grid.md

```diff
@@ -211,7 +218,7 @@ function GridExample() {
   )
 }
 
-<GridExample />
+;<GridExample />
```

Comment:
why do we do this?

Reply:
It's an artifact of react styleguidist as it sometimes gets confused if there's previous stuff before like imports and what not. I think it's actually a dependency of react-styleguidist maybe `acorn-jsx` 🤔